docs/REST-API/04-Rate-Limits.md

```diff
@@ -0,0 +1,82 @@
+---
+tags: [rest-api]
+---
+
+# REST API Rate Limits
+
+<!-- theme: info -->
+> This page describes the rate limits for the PagerDuty REST API. The rate limits for the 
+> PagerDuty Events API are described [here](../../docs/events-API-v2/05-Rate-Limits.md).
+
+<!-- theme: warning -->
+> ### Upcoming Changes to REST API Rate Limits
+>
+> Previously this page read: _The rate limits on our REST API are currently 900 requests per minute across an entire organization._ The actual rate limits were closer to 900 requests per minute per token.
+> The behavior now described on this page is being rolled out during October and November 2023. See the [changelog](https://developer.pagerduty.com/api-reference/f1a95bb9397ba-changelog) for more details.
+
+## What are the limits?
+
+The PagerDuty REST API uses rate limits to provide a consistent experience for all of our customers. The limits depend on the type of authorization token being used. Each PagerDuty App has its own set of limits that are separate from both API keys and other apps.
+
+**Account API Keys**
+
+Each Account API Key is allowed 960 requests per minute.
+
+**User API Keys**
+
+User API Keys act as a PagerDuty User and are limited as such. Each user is allowed 960 requests per minute across all of the user's API keys.
+
+**PagerDuty App - App OAuth Tokens**
+
+When a PagerDuty App [uses an app token](../../docs/app-integration-development/08-OAuth-Functionality.md) it is acting as the PagerDuty App. Each PagerDuty App is allowed 960 requests per minute against each PagerDuty Account it is authorized to access.
+
+**PagerDuty App - User OAuth Tokens**
+
+When a PagerDuty App [uses a user token](../../docs/app-integration-development/08-OAuth-Functionality.md) it is acting as the PagerDuty User. Each PagerDuty App is allowed 960 requests per minute per User it is authorized to act as.
+
+### Operation specific limits
```

Comment:
I think Operation Specific Limits can be at the same level as the above "What are the limits?" question.

Reply:
The limits have two components: 

* The standard / base limits
* Operation specific limits

I added an additional section to try and keep everything organized.